### PR TITLE
minor changes in infrastructure section of docs

### PR DESCRIPTION
**Pull request by prachi237**
PR Checklist:

- [x] make all edits to the docs in the `src` directory, not in `docs` or in the html files
- [x] note any issues closed by this PR with [closing keywords](https://help.github.com/articles/closing-issues-using-keywords)
- [x] put any other relevant information below

### CODE REVIEW

File: src/maintainer/infrastructure.rst [hunk 1]
```diff
@@ -19,11 +19,11 @@ Smithy contains maintenance code for conda-forge, which is used by the ``conda-s
...
 ``conda-forge/conda-smithy`` is the right repository to report bugs for
 
- - the rerendering process
- - the recipe linter
+ - The rerendering process
+ - The recipe linter
  - :term:`CI` support utils
 
-``conda-smithy`` also contains the command line tool that you will use if you rerender manually from the command line (see :ref:`dev_update_rerender`).
+``conda-smithy`` also contains the command line tool that you can use if you rerender manually from the command line (see :ref:`dev_update_rerender`).
```
Comment (jakirkham) [inline src/maintainer/infrastructure.rst:26]: It's the only tool we use to re-render (particularly when doing this locally). So maybe "should" is better

```suggestion
``conda-smithy`` also contains the command line tool that you should use if you rerender manually from the command line (see :ref:`dev_update_rerender`).
```

**Review by viniciusdc — APPROVED** [review]
Lgtm

**Review by jakirkham — COMMENTED** [review]
Looks good. Thanks Prachi! 😄

Had one suggestion above

**Comment by jakirkham** [comment]
Thanks Prachi! 😄

**Pull request merged**